cdap-docs/examples-manual/source/examples/index.rst

```diff
@@ -45,6 +45,9 @@ In addition to the :ref:`Getting Started's <getting-started-index>`
   * - :doc:`File Sets<fileset>`
     - A variation of the *WordCount* example that operates on files. It demonstrates the usage of the FileSet
       dataset, including a service to upload and download files, and a MapReduce that operates over these files.
+  * - :doc:`Log Analysis <log-analysis>`
+    - A Cask Data Application Platform (CDAP) example demonstrating Spark and MapReduce running in parallel
```

Comment:
An example demonstrating Spark and MapReduce running in parallel inside a workflow, showing the use of conditional workflows with forks and branches.